valhalla/jawn/src/lib/proxy/WebSocketProxyForwarder.ts

```diff
@@ -113,70 +197,16 @@ export function webSocketProxyForwarder(
               timestamp: new Date().toISOString(),
               from,
             });
-            /* -------------------------------------------------------------------------- */
-            /*                            Handle Closing Event                            */
-            /* -------------------------------------------------------------------------- */
-          } else if (messageType === "close") {
-            const authClient = getHeliconeAuthClient();
-            try {
-              // 1. Handle the socket session with socket messages
-              const { loggable } = await handleSocketSession(
-                messages,
-                requestWrapper
-              );
-
-              // 2. Get the auth
-              const { data: auth, error: authError } =
-                await requestWrapper.auth();
-              if (authError !== null) {
-                console.error("Error getting auth", authError);
-                return;
-              }
-
-              // 3. Get the auth params
-              const { data: authParams, error: authParamsError } =
-                await authClient.authenticate(auth);
-
-              if (authParamsError || !authParams) {
-                console.error("Error getting auth params", authParamsError);
-                return;
-              }
-
-              // 4. Get the org params
-              const { data: orgParams, error: orgParamsError } =
-                await authClient.getOrganization(authParams);
-
-              if (orgParamsError || !orgParams) {
-                console.error("Error getting organization", orgParamsError);
-                return;
-              }
-
-              // 5. Log the session
-              const result = await loggable.log(
-                {
-                  s3Manager: new S3Manager(
-                    new S3Client(
-                      process.env.S3_ACCESS_KEY ?? "",
-                      process.env.S3_SECRET_KEY ?? "",
-                      process.env.S3_ENDPOINT ?? "",
-                      process.env.S3_BUCKET_NAME ?? "",
-                      (process.env.S3_REGION as "us-west-2" | "eu-west-1") ??
-                        "us-west-2"
-                    )
-                  ),
-                  kafkaProducer: new HeliconeQueueProducer(),
-                },
-                authParams,
-                orgParams,
-                requestWrapper.heliconeHeaders
-              );
-
-              if (result.error) {
-                console.error("Error logging WebSocket session:", result.error);
-              }
-            } catch (error) {
-              console.error("Error handling socket session:", error);
+          }
+          /* -------------------------------------------------------------------------- */
+          /*                            Handle Closing Event                            */
+          /* -------------------------------------------------------------------------- */
```

Comment:
@charliemday and @connortbot  thoughts on these big scoping comments? Was never a big fan, wondering if we should remove them moving forward